index.js

```diff
@@ -1,7 +1,45 @@
-/* eslint-disable no-case-declarations */
+const { Readable } = require('readable-stream');
 const parseDataUrl = require('parse-data-url');
+const { Blob: BlobPolyfill, File: FilePolyfill } = require('formdata-node');
 
-function constructRequest(har, userAgent = false) {
+// Instead of requiring the user to polyfill this on their end we're doing it ourselves so we can ensure that we have
+// an API we know will work.
+if (!globalThis.Blob) {
+  globalThis.Blob = BlobPolyfill;
+}
+
+if (!globalThis.File) {
+  globalThis.File = FilePolyfill;
+}
+
+/**
+ * @license MIT
+ * @see {@link https://github.com/octet-stream/form-data-encoder/blob/master/lib/util/isFunction.ts}
+ */
+function isFunction(value) {
+  return typeof value === 'function';
+}
+
+/**
+ * We're loading this library in here instead of loading it from `form-data-encoder` because that uses lookbehind
+ * regex in its main encoder that Safari doesn't support so it throws a fatal page exception.
```

Comment:
This is the line of code that completely crashes Safari out when `form-data-encoder` is loaded there: https://github.com/octet-stream/form-data-encoder/blob/master/lib/util/normalizeValue.ts#L11